runners/remote-worker/src/lib/credhelper.ts

```
@@ -161,14 +154,14 @@ if [ -n "\$PUBLISHER_CLIENT_ID" ] && [ -n "\$PUBLISHER_CLIENT_SECRET" ] && [ -n
     fi
   fi
   if [ -z "\$bearer" ]; then
-    echo "credhelper: publisher client_credentials mint failed; falling back to \\\$AEP_BEARER_FILE" >&2
+    echo "credhelper: publisher client_credentials mint failed; trying \\\$AEP_BEARER_FILE (CC snapshot)" >&2
```

ESLint

[error] Unnecessary escape character: \$.

(no-useless-escape)